client/src/components/MenueCard.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+const InputContainer = styled.div`
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: center;
+  padding: 2px;
+  margin: 5px 2px;
+  outline: none;
+`;
+
+const Card = styled.div`
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  min-width: 300px;
+  height: auto;
+  margin-top: 0.4rem;
+  border-radius: 1.5rem;
+  color: #323338;
+  box-shadow: 0px 10px 15px rgba(0, 0, 0, 16%);
+`;
+
+const Img = styled.img`
+  width: 300px;
```

Comment:
I recommend moving `width` to the parent element :)

Reply:
I don't know why but the picture will be huge, so that's why I gave the width to the picture itself 😅